WooCommerce/Classes/ViewRelated/Orders/Order Details/OrderDetailsViewController.swift

```diff
@@ -676,6 +676,8 @@ private extension OrderDetailsViewController {
         let editAddressViewController = EditAddressHostingController(viewModel: viewModel)
         let navigationController = WooNavigationController(rootViewController: editAddressViewController)
         present(navigationController, animated: true, completion: nil)
+
+        ServiceLocator.analytics.track(event: WooAnalyticsEvent.OrderDetailsEdit.orderDetailEditFlowStarted(subject: .shippingAddress))
```

Comment:
Can't we move start events to EditAddressFormViewModel `init` or `onLoadTrigger`?

Reply:
I normally don't like to add analytics code there because it is an unwanted side effect that people do not expect.

EG:
- I could create the VC and not present it yet, but if the analytics code is on `init` the event would be fired.


But now that I think about it, since this `onLoadTrigger()` does not follow `UIKit`, there is no chance for this to be double executed if the VC gets evicted from memory.